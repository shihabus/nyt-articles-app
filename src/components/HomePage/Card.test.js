 /* eslint-disable import/named */
/* eslint-disable no-undef */
/**
* @jest-environment jsdom
*/

// import React from "react";
// import { shallow } from "enzyme";
// import { createSerializer } from "enzyme-to-json";
// import Card from "./Card";
// import { findComponent, testStore, checkProps } from "../../../testUtils";

// expect.addSnapshotSerializer(createSerializer({ mode: "deep" }));

// const setUp = (props = {}) => {
//   const wrapper = shallow(<Card {...props} />);
//   console.log(wrapper);
//   return wrapper;
// };

// describe("Home Page Card", () => {
//   let wrapper;
//   beforeEach(() => {
//     const props = {
//       item: {
//         byline: "Test",
//         source: "Test",
//         published_date: "Test",
//         title: "Test",
//         media: [{ id: 1 }, { id: 2 }],
//         id: 10,
//         abstract: "Test",
//         url: "Test"
//       },
//       handleSelection: () => {}
//     };
//     wrapper = setUp(props);
//   });
//   console.log(wrapper)

//   it("Snapshot test", () => {
//     expect(1).toBe(1)
//   });

//   it("Checked props", () => {
//     expect(1).toBe(1)
//   });
// });

it('Dummy',()=>{
  expect(1).toBe(1)
})