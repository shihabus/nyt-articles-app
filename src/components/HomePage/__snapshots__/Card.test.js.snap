// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Card Snapshot test: Card 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "alignContent": "center",
      "backgroundColor": "#fff",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "paddingVertical": 20,
    }
  }
  testID="Card"
>
  <ImageContainer
    testID="ImageContainer"
    urlToImage="https://static01.nyt.com/images/2019/06/27/multimedia/27xp-alabama1/27xp-alabama1-thumbLarge.jpg"
  />
  <TextContainer
    params={
      Object {
        "abstract": "Test",
        "byline": "Test",
        "imageURL": "https://static01.nyt.com/images/2019/06/27/multimedia/27xp-alabama1/27xp-alabama1-mediumThreeByTwo440.jpg",
        "published_date": "Test",
        "source": "Test",
        "title": "Test",
        "url": "Test",
      }
    }
    testID="TextContainer"
  />
  <IconContainer
    testID="IconContainer"
  />
</TouchableOpacity>
`;
